Makefile

```diff
@@ -277,6 +306,10 @@ install: $(addprefix $(BUILDDIR)/,$(INSTALLEES))
 	for prog in $(INSTALLEES); do \
 		install $(BUILDDIR)/$$prog $(DESTDIR)$(PREFIX)$(BINDIR)/$(PROG_PREFIX)$$prog; \
 	done
+	mkdir -p $(DESTDIR)$(PREFIX)$(LIBDIR)
+	for lib in $(LIBS); do \
+		install $(BUILDDIR)/$$lib $(DESTDIR)$(PREFIX)$(LIBDIR)/$$lib; \
+	done
```

Comment:
To get this working for `install-multicall`, I believe you would do the same thing you are doing here.